native/cocos/scene/RenderWindow.cpp

```diff
@@ -143,6 +139,34 @@ void RenderWindow::extractRenderCameras(ccstd::vector<Camera *> &cameras) {
     }
 }
 
+void RenderWindow::onNativeWindowDestroy(uint32_t windowId) {
+    if (_swapchain != nullptr && _swapchain->getWindowId() == windowId) {
+        _swapchain->destroySurface();
+    }
+}
+
+void RenderWindow::onNativeWindowResume(uint32_t windowId) {
+    auto *windowMgr = BasePlatform::getPlatform()->getInterface<ISystemWindowManager>();
+    auto *hWnd = reinterpret_cast<void *>(windowMgr->getWindow(windowId)->getWindowHandle());
+
+    if (_swapchain == nullptr || _swapchain->getWindowId() != 0) {
+        return;
+    }
+    _swapchain->createSurface(hWnd);
+    updateFramebuffer();
+}
+
+void RenderWindow::updateFramebuffer() {
+    if (_frameBuffer != nullptr) {
+        _frameBuffer->destroy();
+        _frameBuffer->initialize({
```

Comment:
we got a convention that GFXObject should not be reinit. 
way of:
```
_frameBuffer->destroy();
_frameBuffer->initialize(info);
...
```
may be changed to
```
_frameBuffer->destroy(); // do not reuse
_frameBuffer = device->createFramebuffer(info);
...
```

Reply:
In cpp, should delete, not just destroy if want to create a new one. And i agree should not reuse gfx object.

Reply:
I agree with that. Gfx objects should not initialize twice.